### PR TITLE
Auto assign only 1 person from list of `persons` or the first team specified in `teams`

### PR DESCRIPTION
**Pull request by michlimlim**
## Changes
Created an optional boolean parameter `pick-one-from-persons-or-team` if users would like for one team member to be picked for assignment for each PR

Also updated the npm packages and used the latest Octokit methods

## Motivation
When we used the original `teams` method, our entire Github reviewers team was notified every time a PR was open, even though we did pick "round robin" and only assigned one reviewer each time. 

Therefore we created a way to only assign one reviewer from a Github team or list of persons at a time. Say no more to your entire team being spammed every time a PR is opened. 

## Testing

- Functionality of `pick-one-from-persons-or-team=true` works when our org is using this forked version. One reviewer is assigned each time from our Github team.
- The original unit tests still pass.
- But I'd love for someone to add a new unit test for `pick-one-from-persons-or-team=true`

### CODE REVIEW

**Comment by rowi1de** [comment]
Hey thanks for the PR. Will review soon.
Regarding the team notification: Did you Check  the box „don’t notify team“?
https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team
https://docs.github.com/assets/images/help/teams/review-assignment-notifications.png

**Review by rowi1de — APPROVED** [review]
Thanks for the contribution, will release a new version after merge

**Comment by michellelimxuanli** [comment]
> Thanks for the contribution, will release a new version after merge

Awesome! Btw for the new setting, I forgot to remove PR author from the Github Team list. Have made another commit and will open a PR again once the current PR is merged.

> Regarding the team notification: Did you Check the box „don’t notify team“?

Yes, we had already checked the box.

**Comment by rowi1de** [comment]
Somehow the tests are failing

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -4,21 +4,21 @@ on:
...
-        include-draft: false
-        skip-with-manual-reviewers: 1
+      - run: npm ci
+      - run: npm run build
+      - run: npm test
+      - uses: ./
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          persons: "rowi1de"
+          teams: "no-team"
```
Comment (rowi1de) [inline .github/workflows/test.yml:22]: this needs to be removed

**Review by rowi1de — CHANGES_REQUESTED** [review]
Please remove team from test.yml

**Comment by rowi1de** [comment]
@michlimlim let's finish this, I recently upgraded to GHA 5.0.0

**Comment by rowi1de** [comment]
https://github.com/rowi1de/auto-assign-review-teams/pull/58

**Pull request closed without merging**